cluster.go

```diff
@@ -589,25 +589,28 @@ func (c *clusterClient) _pickMulti(multi []Completed) (retries *connretry) {
 		} else if init && last != cmd.Slot() {
 			panic(panicMixCxSlot)
 		}
-		p := c.pslots[cmd.Slot()]
-		if p == nil {
+		cc := c.pslots[cmd.Slot()]
+		if cc == nil {
 			return nil
 		}
-		count.m[p]++
+		count.m[cc]++
 	}
 
 	if last == cmds.InitSlot {
 		// if all commands have no slots, such as INFO, we pick a non-nil slot.
-		for i, p := range c.pslots {
-			if p != nil {
+		for i, cc := range c.pslots {
+			if cc != nil {
 				last = uint16(i)
-				count.m[p] = inits
+				count.m[cc] = inits
 				break
 			}
 		}
 		if last == cmds.InitSlot {
 			return nil
 		}
+	} else if init {
+		cc := c.pslots[last]
+		count.m[cc] += inits
```

Comment:
We don't count cmd.InitSlot commands to `c.pslots[last]` in the code above this, so add the count back here.